lisa/tools/interrupt_inspector.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from __future__ import annotations
+
+import re
+from typing import Dict, List
+
+from lisa.executable import Tool
+from lisa.tools import Cat
+
+
+class Interrupt:
+    irq_number: str
+    interrupt_count: List[int]
+    metadata: str
+    interrupt_sum: int
+
+    def __init__(
+        self,
+        irq_number: str,
+        interrupt_count: List[int],
+        interrupt_sum: int,
+        metadata: str = "",
+    ) -> None:
+        self.irq_number = irq_number
+        self.interrupt_count = interrupt_count
```

Comment:
`cpu_counter`. It doesn't need to repeat "interrupt" in the variable name.

Reply:
updated